modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} update process.
+
+. The intent to update the cluster is expressed through the web console or the CLI.
+The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource.
```

Comment:
nit: for this concept-focused module, can we de-emphasize the UIs?  For example:

```asciidoc
. The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource, which may be managed through web console or the CLI.
```

I've removed "the intent to update", because the `spec` property is about where the cluster admin wants the cluster to be (a target level).  The cluster version operator can decide whether it wants to use update-mode or reconcile-mode to achieve that target level, but the admin does not need to think about "is the target I'm about to set set the same as or different from the one the CVO is currently trying to reconcile?".  This is a pretty narrow hair to split, and whether it's worth it may depend on [whether you address non-update CVO modes][1].  But for this particular list entry, "The target version" seems like a more approachable concept to me than "The intent to update".

One approach that can  be useful to limit repeat bikeshedding is lifting wording from [the API docs][2] for the property being documented.  Although I'm happy to bikeshed, or to be told to just mind my own business while other folks pick docs wording, or whatever ;).

[1]: https://github.com/openshift/openshift-docs/pull/60091#discussion_r1212377474
[2]: https://github.com/openshift/api/blob/2346bf94f19e7a325340c8dd65530d18c4027079/config/v1/types_cluster_version.go#L46-L69

Reply:
I'll go with your suggested wording here